### PR TITLE
A Clock With Benefits

### PR DESCRIPTION
**Pull request by tomburtonwood**
i'm worried my proposals might be too complex for the actual project. but these are the top four projects i want to work on over the semester.

### CODE REVIEW

**Comment by bakercp** [comment]
Hey @tomburtonwood this looks great.  A few little website structure things ... since you're the first, let's move the images folder to the root of the website -- you can do that by changing the directory to the `_posts` folder and running:

`git mv images/ ../` 

That will move the images to the root folder.

Then, when you want to reference them in a post, use the following code:

```html
<img src="{{ site.baseurl }}/images/enzo.png" width="600px" />
```

or for pure markdown (if you don't care about image size).

```md
![]({{ site.baseurl }}/images/enzo.png)
```

The `{{ site.baseurl }}` will grab the base URL from the config and fill it in correctly.

**Comment by tomburtonwood** [comment]
cool - yeah i was having a hard time getting the image markdown code to
work - but in retrospect scaling the imgs down is prob a good idea, though
i guess i could reformat them in terminal - so .... maybe i'll convert to
markdown after all.



On Sun, Sep 13, 2015 at 8:31 PM, Christopher Baker <notifications@github.com
> wrote:

> Hey @tomburtonwood <https://github.com/tomburtonwood> this looks great. A
> few little website structure things ... since you're the first, let's move
> the images folder to the root of the website -- you can do that by changing
> the directory to the _posts folder and running:
>
> git mv images/ ../
>
> That will move the images to the root folder.
>
> Then, when you want to reference them in a post, use the following code:
>
> 
>
> or for pure markdown (if you don't care about image size).
>
> 
>
> The {{ site.baseurl }} will grab the base URL from the config and fill it
> in correctly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139936862>
> .
>



-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

**Comment by bakercp** [comment]
If you want to resize images on the command line -- just for the fun of it, `sips` or image magick are good (http://www.maclife.com/article/columns/terminal_101_resize_images_terminal).  Smaller images in git repositories are usually preferred.

**Comment by bakercp** [comment]
Ahh, one other thing -- you need to prefix your post filename with the date.  You can fix yours like this if you are in the `_posts` directory:

```
git mv Burtonwood-ClockWithBenefits.md 2015-09-13-Burtonwood-ClockWithBenefits.md
```

**Comment by tomburtonwood** [comment]
will do. i tried to use git last year to publish my last 3DP book and i
uploaded, or tried to, probably 100mb in STL's did not work very well :(

On Sun, Sep 13, 2015 at 8:37 PM, Christopher Baker <notifications@github.com
> wrote:

> If you want to resize images on the command line -- just for the fun of
> it, sips or image magick are good (
> http://www.maclife.com/article/columns/terminal_101_resize_images_terminal).
> Smaller images in git repositories are usually preferred.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139937729>
> .
>



-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

**Comment by tomburtonwood** [comment]
got it. i did not know if that was auto generated.


On Sun, Sep 13, 2015 at 8:39 PM, Christopher Baker <notifications@github.com
> wrote:

> Ahh, one other thing -- you need to prefix your post filename with the
> date. You can fix yours like this if you are in the _posts directory:
>
> git mv Burtonwood-ClockWithBenefits.md 2015-09-13-Burtonwood-ClockWithBenefits.md
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139937852>
> .
>



-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

**Comment by tomburtonwood** [comment]
is there a way to git add all updated files automatically rather than
typing in all the files names?

On Sun, Sep 13, 2015 at 8:40 PM, tom burtonwood <tburtonwood@gmail.com>
wrote:

> got it. i did not know if that was auto generated.
>
>
> On Sun, Sep 13, 2015 at 8:39 PM, Christopher Baker <
> notifications@github.com> wrote:
>
>> Ahh, one other thing -- you need to prefix your post filename with the
>> date. You can fix yours like this if you are in the _posts directory:
>>
>> git mv Burtonwood-ClockWithBenefits.md 2015-09-13-Burtonwood-ClockWithBenefits.md
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139937852>
>> .
>>
>
>
>
> --
>
> tom burtonwood
> tburtonwood@gmail.com
>
> http://tomburtonwood.com
> http://learn3dprinting.co
>
> phone: 773-837-9809
> twitter: @tburtonwood
> Skype: tburtonwood
> instagram: @tomburtonwood
>
> Need 3D Printing? Check out our 3D Hub:
> https://www.3dhubs.com/chicago/hubs/simile
>
>


-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

**Comment by tomburtonwood** [comment]
not sure why imgs are not showing up now
https://github.com/tomburtonwood/InteractiveArt2015/blob/gh-pages/_posts/2015-09-13-Burtonwood-ClockWithBenefits.md

i changed images to _images and changed the markdown to reflect this - but
nope.


On Sun, Sep 13, 2015 at 8:49 PM, tom burtonwood <tburtonwood@gmail.com>
wrote:

> is there a way to git add all updated files automatically rather than
> typing in all the files names?
>
> On Sun, Sep 13, 2015 at 8:40 PM, tom burtonwood <tburtonwood@gmail.com>
> wrote:
>
>> got it. i did not know if that was auto generated.
>>
>>
>> On Sun, Sep 13, 2015 at 8:39 PM, Christopher Baker <
>> notifications@github.com> wrote:
>>
>>> Ahh, one other thing -- you need to prefix your post filename with the
>>> date. You can fix yours like this if you are in the _posts directory:
>>>
>>> git mv Burtonwood-ClockWithBenefits.md 2015-09-13-Burtonwood-ClockWithBenefits.md
>>>
>>> —
>>> Reply to this email directly or view it on GitHub
>>> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139937852>
>>> .
>>>
>>
>>
>>
>> --
>>
>> tom burtonwood
>> tburtonwood@gmail.com
>>
>> http://tomburtonwood.com
>> http://learn3dprinting.co
>>
>> phone: 773-837-9809
>> twitter: @tburtonwood
>> Skype: tburtonwood
>> instagram: @tomburtonwood
>>
>> Need 3D Printing? Check out our 3D Hub:
>> https://www.3dhubs.com/chicago/hubs/simile
>>
>>
>
>
> --
>
> tom burtonwood
> tburtonwood@gmail.com
>
> http://tomburtonwood.com
> http://learn3dprinting.co
>
> phone: 773-837-9809
> twitter: @tburtonwood
> Skype: tburtonwood
> instagram: @tomburtonwood
>
> Need 3D Printing? Check out our 3D Hub:
> https://www.3dhubs.com/chicago/hubs/simile
>
>


-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

File: _posts/2015-09-13-Burtonwood-ClockWithBenefits.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title:  "Tom Burtonwood: A Clock With Benefits, 4 proposals"
+date:   2015-09-13 00:00:00
+categories: post
+author: tomburtonwood
+---
+
+![]({{ site.baseurl }}_images/enzo.png)
```
Comment (bakercp) [inline _posts/2015-09-13-Burtonwood-ClockWithBenefits.md]: I believe these should be:

```
![]({{ site.baseurl }}/images/enzo.png)
```

**Comment by bakercp** [comment]
Also, can you put them in a folder called "images" rather than "_images".  I think the underscore causes them to not get added to the site as assets.

File: _posts/2015-09-13-Burtonwood-ClockWithBenefits.md (hunk 1 above)
Comment (bakercp) [inline _posts/2015-09-13-Burtonwood-ClockWithBenefits.md]: Also you'll need to put them in a folder called `images` rather than `_images`.

**Comment by bakercp** [comment]
Regarding adding all changes ... generally it's a good idea to be specific, though you can use wildcards like *.  The problem there is that it will often add stuff you don't want to ... like `.DS_Store` files, etc if you aren't ignoring them with the `.gitignore` file.

File: _data/people/tomburtonwood.yml.save [hunk 2]
```diff
@@ -0,0 +1,8 @@
+user: tomburtonwood
```
Comment (bakercp) [inline _data/people/tomburtonwood.yml.save]: I think this file should be removed ...

File: _data/people/tomburtonwood.yml.save.1 [hunk 3]
```diff
@@ -0,0 +1,8 @@
+user: tomburtonwood
```
Comment (bakercp) [inline _data/people/tomburtonwood.yml.save.1]: I think this file should be removed too ...

File: _data/people/tomburtonwood.yml.save (hunk 2 above)
Comment (tomburtonwood) [inline _data/people/tomburtonwood.yml.save]: eeek. sorry. my bad.

On Sun, Sep 13, 2015 at 9:34 PM, Christopher Baker <notifications@github.com
> wrote:

> In _data/people/tomburtonwood.yml.save
> <https://github.com/bakercp/InteractiveArt2015/pull/21#discussion_r39357643>
> :
>
> > @@ -0,0 +1,8 @@
> > +user: tomburtonwood
>
> I think this file should be removed ...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bakercp/InteractiveArt2015/pull/21/files#r39357643>.
>



-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

Comment (bakercp) [inline _data/people/tomburtonwood.yml.save]: Note that it is ... `yml.save`

File: _data/people/tomburtonwood.yml.save.1 (hunk 3 above)
Comment (bakercp) [inline _data/people/tomburtonwood.yml.save.1]: Note that it is ... `yml.save.1`

Comment (tomburtonwood) [inline _data/people/tomburtonwood.yml.save.1]: ok i think everything is back to as it should be.

imgs still not showing up though. -

On Sun, Sep 13, 2015 at 9:35 PM, Christopher Baker <notifications@github.com
> wrote:

> In _data/people/tomburtonwood.yml.save.1
> <https://github.com/bakercp/InteractiveArt2015/pull/21#discussion_r39357669>
> :
>
> > @@ -0,0 +1,8 @@
> > +user: tomburtonwood
>
> Note that it is ... yml.save.1
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bakercp/InteractiveArt2015/pull/21/files#r39357669>.
>



-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

File: _posts/2015-09-13-Burtonwood-ClockWithBenefits.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+application / website: Structure scanner export application and web interface / web project to display / view scans, with later options to categorize and organize the scans and allow visitors to sort them. the application would allow a user to upload obj files from existing structure scanner app / SDK on ios and see them loaded up into a 3d space in the web browser that they can share with others. This example ( <https://sketchfab.com/models/e6b8e2b5c19945ee813504253a34c8c7?utm_source=oembed&utm_medium=embed&utm_campaign=e6b8e2b5c19945ee813504253a34c8c7>) - on sketchfab is ok but aesthetically i'm looking for something more stripped down and capable of showing many obj files at once.
+
+![]({{ site.baseurl }}/images/structure.jpeg)
+
+**2. Improve resolution / range of Structure ios scanner.**
+
+software/app: To build my own scanner using a kinect / structure, to try to improve the resolution  and get better results, either larger scanned areas, increasing the focal length, to scan from a drone in realtime for example, or to scan larger structures with better resolution results, than currently available
+
+![]({{ site.baseurl }} /images/sullivan.JPG)
```
Comment (bakercp) [inline _posts/2015-09-13-Burtonwood-ClockWithBenefits.md]: remove the space between `}} /images`

File: _posts/2015-09-13-Burtonwood-ClockWithBenefits.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+**3. Augmented Reality app for upcoming 3DP book**
+
+software/app: Augmented Reality Application. For upcoming Sullivan book, i want to make an AR app to provide access to extra layers on content that cannot easily be “displayed” on the 3D printed or physical pages. This comes out of a long standing interest in embedding scannable symbols or trackers in 3d printed or digitally fabricated pieces - it would be awesome to figure out a way to do this in monochrome - so one color plastic extrusion, or to automatically generate a smaller icon that fits or is inlaid into the object - or that the object itself is a symbol / tracker / trigger for the AR app. i’m fascinated by the idea that the object is a physical form of url - and that this stream of info can be variable - and be updated over time.
+
+![]({{ site.baseurl }} /images/madlion-mold.png)
```
Comment (bakercp) [inline _posts/2015-09-13-Burtonwood-ClockWithBenefits.md]: again, remove the space.  should look like:

``

**Comment by bakercp** [comment]
So close!

**Comment by tomburtonwood** [comment]
sorry. i missed those. they were my tests. are you seeing any of my images
at that end? i don't see any at this end. will update.

On Sun, Sep 13, 2015 at 10:10 PM, Christopher Baker <
notifications@github.com> wrote:

> So close!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139949574>
> .
>



-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

**Comment by bakercp** [comment]
Looks like it's all working now ...

http://tomburtonwood.github.io/InteractiveArt2015/post/2015/09/12/Burtonwood-ClockWithBenefits.html

**Comment by tomburtonwood** [comment]
aha! i'm looking at my github page - the imgs are not showing there. i did
not think to look on the class website. any thoughts why they show up
there, but not here


https://github.com/tomburtonwood/InteractiveArt2015/blob/gh-pages/_posts/2015-09-13-Burtonwood-ClockWithBenefits.md
?


On Sun, Sep 13, 2015 at 10:17 PM, Christopher Baker <
notifications@github.com> wrote:

> Looks like it's all working now ...
>
>
> http://tomburtonwood.github.io/InteractiveArt2015/post/2015/09/12/Burtonwood-ClockWithBenefits.html
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139951402>
> .
>



-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

**Comment by tomburtonwood** [comment]
thank you for your help btw!

On Sun, Sep 13, 2015 at 10:19 PM, tom burtonwood <tburtonwood@gmail.com>
wrote:

> aha! i'm looking at my github page - the imgs are not showing there. i did
> not think to look on the class website. any thoughts why they show up
> there, but not here
>
>
> https://github.com/tomburtonwood/InteractiveArt2015/blob/gh-pages/_posts/2015-09-13-Burtonwood-ClockWithBenefits.md
> ?
>
>
> On Sun, Sep 13, 2015 at 10:17 PM, Christopher Baker <
> notifications@github.com> wrote:
>
>> Looks like it's all working now ...
>>
>>
>> http://tomburtonwood.github.io/InteractiveArt2015/post/2015/09/12/Burtonwood-ClockWithBenefits.html
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139951402>
>> .
>>
>
>
>
> --
>
> tom burtonwood
> tburtonwood@gmail.com
>
> http://tomburtonwood.com
> http://learn3dprinting.co
>
> phone: 773-837-9809
> twitter: @tburtonwood
> Skype: tburtonwood
> instagram: @tomburtonwood
>
> Need 3D Printing? Check out our 3D Hub:
> https://www.3dhubs.com/chicago/hubs/simile
>
>


-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

**Comment by bakercp** [comment]
Yeah, they don't show up there because the correct links are not generated until it gets turned into a website on upload.  Note that http://tomburtonwood.github.io/InteractiveArt2015/post/2015/09/12/Burtonwood-ClockWithBenefits.html is _your_ version of the site, not the class'.  Now I'll pull your changes in and you will see your additions in the overall class site.

http://bakercp.github.io/InteractiveArt2015/post/2015/09/12/Burtonwood-ClockWithBenefits.html

**Comment by bakercp** [comment]
Thanks!

**Comment by tomburtonwood** [comment]
awesome. thank you - i wanted to get everything online sooner than later,
tomorrow is week 1 of my sculpture class and i still have a lecture to
finish for Weds night. i am sorry to miss Weds class, but i'm hoping to do
the ofSketch graphics work tomorrow or tues night.

t

On Sun, Sep 13, 2015 at 10:21 PM, Christopher Baker <
notifications@github.com> wrote:

> Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139951674>
> .
>



-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

**Comment by bakercp** [comment]
Super.  Thanks for your hard work getting all of that up.

**Comment by tomburtonwood** [comment]
hey - that img resize link u sent is great. for years i've been wanting to
do that without having to open preview or psd - now i want to make an app
to tag and organize all my photos as i take them on my phone.

On Sun, Sep 13, 2015 at 10:28 PM, Christopher Baker <
notifications@github.com> wrote:

> Super. Thanks for your hard work getting all of that up.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bakercp/InteractiveArt2015/pull/21#issuecomment-139952787>
> .
>



-- 

tom burtonwood
tburtonwood@gmail.com

http://tomburtonwood.com
http://learn3dprinting.co

phone: 773-837-9809
twitter: @tburtonwood
Skype: tburtonwood
instagram: @tomburtonwood

Need 3D Printing? Check out our 3D Hub:
https://www.3dhubs.com/chicago/hubs/simile

**Pull request merged**